### PR TITLE
Feb dependencies update and audit fixes

### PR DESCRIPTION
**Pull request by vinesh-ghela**
Before submitting my PR to Code Review, I have made sure:

- [x] I have run the tests using `yarn run test`
- [x] I have exported any new files in index.js
- [ ] My file supports Formik if will potentially be used in a form
- [ ] The PR name follows the convention `(#<ticket_number>) - <ticket_user_story>`
- [x] I have incremented the version in `package.json` if I want to publish a new version
- [ ] If my change is breaking I have updated the minor version number(i.e. the middle one) and don't merge until I have the PR on Corazon ready

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,8 +13,8 @@
     "@storybook/addons": "5.3.0",
```
Comment (RaquelCasadoCRC) [inline package.json:13]: Bump version

File: yarn-audit.html [hunk 2]
```diff
@@ -0,0 +1,1610 @@
+<!doctype html>
```
Comment (RaquelCasadoCRC) [inline yarn-audit.html:1]: Is this the audit report? Can we not commit this?

File: yarn.lock [hunk 3]
```diff
@@ -2,10 +2,17 @@
...
+  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.1.1.tgz#7922fb0817bf3166d8d9e258c57477e3fd1c3610"
+  integrity sha512-Aolwjd7HSC2PyY0fDj/wA/EimQT4HfEnFYNp5s9CQlrdhyvWTtvZ5YzrUPu6R6/1jKiUlxu8bUhkdSnKHNAHMA==
+  dependencies:
+    "@jridgewell/trace-mapping" "^0.3.0"
+
 "@babel/cli@^7.1.5":
-  version "7.15.7"
-  resolved "https://registry.yarnpkg.com/@babel/cli/-/cli-7.15.7.tgz#62658abedb786d09c1f70229224b11a65440d7a1"
-  integrity sha512-YW5wOprO2LzMjoWZ5ZG6jfbY9JnkDxuHDwvnrThnuYtByorova/I0HNXJedrUfwuXFQfYOjcqDA4PU3qlZGZjg==
+  version "7.17.0"
```
Comment (RaquelCasadoCRC) [inline yarn.lock]: Noticed you rolled back the babel update. You probably want to run yarn again to update yarn.lock, build will fail if versions from package.json and yarn.lock are different

File: package.json (hunk 1 above)
Comment (RaquelCasadoCRC) [reply package.json:13]: Design system version xD (I can't comment in the right line)

Comment (vinesh-ghela) [reply package.json:13]: Done

**Review by frank-shenton — APPROVED** [review]

**Pull request merged**